src/NetEscapades.EnumGenerators/SourceGenerationHelper.cs

```diff
@@ -160,9 +160,22 @@ public static string ToStringFast(this ").Append(fullyQualifiedName).Append(@" v
                 .Append(" => nameof(").Append(fullyQualifiedName).Append('.').Append(member.Key).Append("),");
         }
 
-        sb.Append(@"
+        if (enumToGenerate.HasFlags)
+        {
+            // We currently don't handle ToString of custom flag-combinations, so lets fall back to the default
+            sb.Append(@"
                 _ => value.ToString(),
-            };
+            };");
+        }
+        else
+        {
+            // This should mean, that the value is not named -> generate a numeric string
+            sb.Append(@"
+                _ => value.AsUnderlyingType().ToString(),
```

Comment:
I wonder, should this `ToString()` use the current culture, or the `InvariantCulture` 🤔

Reply:
I believe using the current culture would be the drop-in-case. And additionally it does allow the caller to change the culture - even if a bit cumbersome. If you think `InvariantCulture` is the better call, I'll adjust this.

Reply:
According to the answers to https://stackoverflow.com/q/8492449 the negative-sign is culture-dependent, but not much else for the default/`G`-format

Reply:
AFAICT, the underlying `Enum.ToString()` just uses the ambient culture too, so what you've done makes the most sense I think, thanks 🙂